mne/minimum_norm/time_frequency.py

```diff
@@ -442,6 +599,7 @@ def source_induced_power(
     baseline_mode="logratio",
     pca=True,
     n_jobs=None,
+    return_plv=True,
```

Comment:
technically this is an API change, because you're inserting the new param somewhere other than the end of the list, and there is no `*` element in the params indicating the start of "keyword-only" params.  Practically speaking, this is so deep in a long params list that it probably doesn't matter.

My instinct is to go ahead and add in the `*` (after `freqs`?) and then put `return_plv` wherever makes the most sense. @larsoner WDYT?

Reply:
Thanks for the review @drammock ! I think I've addressed the review suggestions now except let me know if you have feedback about this one @larsoner  I'm still working on the tutorial though, so let me get in the tutorial commit and then we should be able to have a final look.